Simulations/python/raw_script.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+x#!/usr/bin/env python3
```

Comment:
I think this `x` is here by accident.
```suggestion
#!/usr/bin/env python3
```
I usually do a three-step process before committing anything:
- First do a `git diff` to inspect all the changes
- Then `git add` the files to commit
- Then a separate `git commit` without `-m` to inspect which files I'm actually going to commit.

And even then it is quite easy to make mistakes. (Well, hence the PR part.)